activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -140,11 +140,14 @@ def humanize(lower_case_and_underscored_word, options = {})
       result
     end
 
-    # Converts just the first character to uppercase.
+    # Converts just the first character to uppercase,
+    # if method called to an empty string, return empty one, too.
     #
-    #  'what a Lovely Day'.upcase_first # => "What a Lovely Day"
+    #   'what a Lovely Day'.upcase_first # => "What a Lovely Day"
+    #   'w'.upcase_first                   => "W"
+    #   ''.upcase_first                    => ""
```

Comment:
@rafaelfranca, this should be `upcase_first('what a Lovely Day')` instead of` 'what a Lovely Day'.upcase_first` ?